recipes/pytest/build.bat

```diff
@@ -0,0 +1,6 @@
+%PYTHON% setup.py install --old-and-unmanageable
```

Comment:
Could we please change `--old-and-unmanageable` to ` --single-version-externally-managed --record=record.txt`?